projects/cdk/schematics/ng-update/v3-30/constants/templates.ts

```diff
@@ -0,0 +1,9 @@
+import {ReplaceableAttributeValue} from '../../v3/interfaces/replaceable-attribute-value';
+import {ICONS} from './icons';
+
+export const REPLACE_ATTR_VALUE: ReplaceableAttributeValue[] = [
+    {
+        attrNames: [`icon`, `src`],
```

Comment:
This should be any attribute, because people may pass icons to their own wrappers with their own input names.